### DISCUSSION TITLE
Custom Commands

### DISCUSSION THREAD

**Discussion by Guuku** [jagrosh/MusicBot#835] [Questions]
Is it possible to use custom commands?  I don't mean aliases, I mean like if I wanted to play a specific video, I could have a command that will play that particular video.  I think this might be what the transforms are but I can't find any documentation on how those work.

**Comment by MichailiK** [comment]
Transforms are experimental, but do allow such a use-case. A suggested alternative would be #483.

**Reply by Guuku** [reply]
It looks like #483 is basically what I'm asking for but it seems like it's not a feature yet unless I'm misunderstanding that thread.

**Reply by MichailiK** [reply]
Yes, #483 is not an implemented feature at all. Transforms are currently an experimental feature, and may get removed in the future. If not, then they'll probably get documented at some point.
